partner_risk_insurance/models/account_move.py

```diff
@@ -36,14 +36,15 @@ class AccountMove(models.Model):
     @api.depends("partner_id", "move_type")
     def _compute_insured_with_credit_policy(self):
         """Compute the insured_with_credit_policy field when partner changes."""
-        self.update(
+        self_w_ctx = self.with_context(check_move_validity=False)
```

Comment:
¿Why change ths instead of add `check_move_validity` in dictionary?

Reply:
Without this context, [account_payment_return_financial_risk](https://github.com/OCA/credit-control/tree/16.0/account_payment_return_financial_risk) module fails when running tests.

`account_payment_return_financial_risk` depends on [account_payment_return](https://github.com/OCA/account-payment/tree/16.0/account_payment_return) that creates move lines [one by one](https://github.com/OCA/account-payment/blob/16.0/account_payment_return/models/payment_return.py#L185-L187) (whitout the counterpart) and making a trigger on `_compute_insured_with_credit_policy` causing the `self.update(...)` fails because the move is not balanced when adding the lines.

Since this module is modifying the `account.move` itself and not the lines nor amounts, I've considered safe to skip move validations and let the other module tests run without issues.